package.json

```diff
@@ -116,5 +118,8 @@
     "shortid": "^2.2.15",
     "systeminformation": "^4.26.9",
     "uuid": "^8.1.0"
+  },
+  "resolutions": {
+    "graphql": "^14.5.8"
```

Comment:
This is yarn specific right?

What issue were you hitting?

Reply:
Yeah this was yarn specific. Removing.